instrumentation/camel-2.20/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/apachecamel/aws/AwsSpanAssertions.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.apachecamel.aws;
+
+import static io.opentelemetry.api.common.AttributeKey.longKey;
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+import static io.opentelemetry.api.trace.SpanKind.CLIENT;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.equalTo;
+import static io.opentelemetry.sdk.testing.assertj.OpenTelemetryAssertions.satisfies;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.sdk.testing.assertj.SpanDataAssert;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+
+class AwsSpanAssertions {
+
+  private AwsSpanAssertions() {}
+
+  static SpanDataAssert sqs(
+      SpanDataAssert span,
+      int index,
+      String spanName,
+      String queueUrl,
+      String queueName,
+      SpanKind spanKind,
+      SpanData parentSpan) {
+    if (index == 0) {
+      span.hasNoParent();
+    } else {
+      span.hasParent(parentSpan);
+    }
+
+    return span.hasName(spanName)
+        .hasKind(spanKind)
+        .hasAttributesSatisfyingExactly(
+            equalTo(stringKey("aws.agent"), "java-aws-sdk"),
+            satisfies(stringKey("aws.endpoint"), val -> val.isInstanceOf(String.class)),
+            equalTo(SemanticAttributes.HTTP_STATUS_CODE, 200),
+            satisfies(
+                stringKey("aws.queue.name"),
+                val ->
+                    val.satisfiesAnyOf(
+                        v -> assertThat(v).isEqualTo(queueName), v -> assertThat(v).isNull())),
+            satisfies(
+                stringKey("aws.queue.url"),
+                val ->
+                    val.satisfiesAnyOf(
+                        v -> assertThat(v).isEqualTo(queueUrl), v -> assertThat(v).isNull())),
+            equalTo(SemanticAttributes.HTTP_METHOD, "POST"),
+            satisfies(
+                SemanticAttributes.HTTP_RESPONSE_CONTENT_LENGTH,
```

Comment:
I think the original was also checking `"http.response_content_length"`. Can you add that as well?

Reply:
thanks for catching this, I think you meant `http.request_content_length`, which I have now added, but let me know if that's not what you meant

Reply:
Yes <derp> thank you!